scripts/tests/panic_on_invalid_genesis.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+set -ex
+# Setup env
+export TORII_P2P_ADDR='127.0.0.1:1341'
+export TORII_API_URL='127.0.0.1:8084'
+export TORII_TELEMETRY_URL='127.0.0.1:8184'
+export IROHA_PUBLIC_KEY='ed01201c61faf8fe94e253b93114240394f79a607b7fa55f9e5a41ebec74b88055768b'
+export IROHA_PRIVATE_KEY='{"digest_function": "ed25519", "payload": "282ed9f3cf92811c3818dbc4ae594ed59dc1a2f78e4241e31924e101d6b1fb831c61faf8fe94e253b93114240394f79a607b7fa55f9e5a41ebec74b88055768b"}'
+export IROHA2_CONFIG_PATH="configs/peer/config.json"
+export SUMERAGI_TRUSTED_PEERS='[{"address":"127.0.0.1:1341", "public_key": "ed01201c61faf8fe94e253b93114240394f79a607b7fa55f9e5a41ebec74b88055768b"}]'
+# Create tmp file for genesis
+export IROHA2_GENESIS_PATH="$(mktemp)" 
+# Remove on exit
+trap 'rm -- "$IROHA2_GENESIS_PATH"' EXIT
```

Comment:
This is a temporary file. It should be cleaned up regardless of exit code.

Reply:
I tested it on my local machine and there is no auto clean up for temporary files. 

![](https://i.imgur.com/ns8snwh.png)